kate/src/lib.rs

```diff
@@ -25,7 +27,7 @@ pub mod config {
 #[cfg(feature = "std")]
 pub mod com;
 
-#[cfg(feature = "alloc")]
+#[cfg(feature = "std")]
```

Comment:
Maybe we could have a "testnet" feature, so that we don't have unnecesary compilation of the tesnet mod unless really needed...

Reply:
Good idea, let me add it to this PR